models/news.models.js

```diff
@@ -36,3 +27,18 @@ exports.findArticles = (sort_by = "created_at") => {
     return articles;
   });
 };
+
+exports.findCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      `SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.article_id FROM comments WHERE article_id = $1 ORDER BY comments.created_at DESC;`,
+      [article_id]
+    )
+    .then(({ rows }) => {
+      const articles = rows;
```

Comment:
The naming of this variable is misleading, are you getting articles or comments?